prow-configurations/README.md

```diff
@@ -0,0 +1,40 @@
+# Prow Installation
+
+## Overview
+
+This folder contains the installation script and the set of configurations for Prow. 
+
+## Prerequisites
+
```

Comment:
Pls add an introductory sentence above the bullet points: `Install the following tools:`